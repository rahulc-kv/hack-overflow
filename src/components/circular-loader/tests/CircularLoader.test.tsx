import React from 'react';
import renderer from 'react-test-renderer';

import Circularloader from '../CircularLoader';

test('Full-screen Circularloader Rendered', () => {
  const component = renderer
    .create(
      <div>
        <Circularloader isFullScreen={false} />
      </div>
    )
    .toJSON();
  expect(component).toMatchSnapshot();
});
