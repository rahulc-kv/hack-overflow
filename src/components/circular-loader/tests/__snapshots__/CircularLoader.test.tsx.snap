// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Full-screen Circularloader Rendered 1`] = `
<div>
  <div
    className="flex justify-center items-center h-[500px]"
  >
    <img
      alt="loader"
      className="w-[80px] h-[80px]"
      src="/images/fullScreenLoader.gif"
    />
  </div>
</div>
`;
